src/bin/cargo/commands/doc.rs

```diff
@@ -17,6 +17,10 @@ pub fn cli() -> App {
         )
         .arg(opt("no-deps", "Don't build documentation for dependencies"))
         .arg(opt("document-private-items", "Document private items"))
+        .arg(opt(
+            "scrape-examples",
+            "Scrape examples from examples/ directory to include as function documentation",
+        ))
```

Comment:
This should be an unstable option to start. Not sure how to do that in cargo.

Reply:
Fixed in 9c9f867